<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CultrueThingsNamerClass</class>
 <widget class="QMainWindow" name="CultrueThingsNamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CultrueThingsNamer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>完全隨機</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>610</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="base_Layout_Tab_1">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_1">
         <item>
          <widget class="QComboBox" name="rmstab_comboBox_CultureOptn">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rmstab_comboBox_RuleOptn">
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_2">
         <item>
          <widget class="QTextBrowser" name="rmstab_result_textBrowser"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_3">
         <item>
          <widget class="QPushButton" name="rmstab_button_SpawnOptn">
           <property name="text">
            <string>Spawn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rmstab_button_CopyOptn">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rmstab_comboBox_CopyPhaseOptn"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>部分隨機</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="base_Layout_Tab_2">
       <item>
        <layout class="QHBoxLayout" name="rmpstab_Layout_Row_1">
         <item>
          <widget class="QComboBox" name="rmpstab_comboBox_CultureOptn"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmpstab_Layout_Row_2">
         <item>
          <layout class="QVBoxLayout" name="rmpstab_Layout_Row_2">
           <item>
            <widget class="QComboBox" name="rmpstab_result_comboBox_1"/>
           </item>
           <item>
            <widget class="QCheckBox" name="rmpstab_ENMode_checkBox_1">
             <property name="text">
              <string>ENMode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rmpstab_Layout_Row_2">
           <item>
            <widget class="QComboBox" name="rmpstab_result_comboBox_2"/>
           </item>
           <item>
            <widget class="QCheckBox" name="rmpstab_ENMode_checkBox_2">
             <property name="text">
              <string>ENMode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rmpstab_Layout_Row_2">
           <item>
            <widget class="QComboBox" name="rmpstab_result_comboBox_3"/>
           </item>
           <item>
            <widget class="QCheckBox" name="rmpstab_ENMode_checkBox_3">
             <property name="text">
              <string>ENMode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmpstab_Layout_Row_3">
         <item>
          <widget class="QLineEdit" name="rmpstab_result_lineEd_1">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rmpstab_result_lineEd_2">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rmpstab_result_lineEd_3">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_4">
         <item>
          <widget class="QPushButton" name="rmpstab_button_spawnOptn">
           <property name="text">
            <string>Spawn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rmpstab_button_copyOptn">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>人物名稱</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>610</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="base_Layout_Tab_3">
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_1">
         <item>
          <widget class="QComboBox" name="rmcanmtab_comboBox_CultureOptn">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rmcanmtab_comboBox_RuleOptn">
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_2">
         <item>
          <widget class="QTextBrowser" name="rmcanmtab_result_textBrowser"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rmcanmtab_Layout_Row_3">
         <item>
          <widget class="QPushButton" name="rmcanmtab_button_SpawnOptn">
           <property name="text">
            <string>Spawn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rmcanmtab_button_CopyOptn">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rmcanmtab_comboBox_CopyPhaseOptn"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CultrueThingsNamer.qrc"/>
 </resources>
 <connections/>
</ui>
